model_evaluation/model_evaluation.py

```diff
@@ -0,0 +1,49 @@
+from sklearn.metrics import f1_score
+from sklearn.metrics import confusion_matrix
+import matplotlib.pyplot as mp
+from model_training import *
```

Comment:
Try to avoid this pattern whenever possible (from <pkg> import *). It makes code harder to read. (that python guide mentions this with some explanation)